nf_core/pipeline-template/.github/workflows/ci.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         NXF_VER:
           - "21.10.3"
           - "latest-everything"
+        profile: ["docker"]
```

Comment:
Shouldn't this also include `test` as we require this as the bare minimum? And shouldn't `docker` be in the command itself by default and the matrix only contains the actual test profiles?

Reply:
The `test` profile is baked into nf-test now! In the matrix, it'll also include singularity and conda